scripts/infra

```diff
@@ -24,13 +24,17 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [[ -n "${GT_SITE_SETTINGS_BUCKET}" ]]; then
         pushd "${DIR}/../terraform"
 
+        aws s3 cp "s3://${GT_SITE_SETTINGS_BUCKET}/terraform/core/terraform.tfvars" \
```

Comment:
We should add the corresponding `rm` to `.travis.yml` in a `after_deploy` block.